### ISSUE TITLE
The 'EFCorePowerToolsPackage' package did not load correctly

### ISSUE THREAD

**Issue by vinayvhub** [ErikEJ/EFCorePowerTools#185]
After the installation of this extension, When I restarted my Visual Studio, I got an error, it pointed me to an error log. Below is the exception. Unable to use any of the features of this extension.

CreateInstance failed for package [EFCorePowerToolsPackage]Source: 'mscorlib' Description: Exception has been thrown by the target of an invocation. System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.IO.FileNotFoundException: Could not load file or assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies. The system cannot find the file specified.   at EFCorePowerTools.EFCorePowerToolsPackage.CreateServiceProvider()   at EFCorePowerTools.EFCorePowerToolsPackage..ctor() in C:\projects\efcorepowertools\src\GUI\EFCorePowerTools\EFCorePowerToolsPackage.cs:line 64   --- End of inner exception stack trace ---   at System.RuntimeTypeHandle.CreateInstance(RuntimeType type, Boolean publicOnly, Boolean noCheck, Boolean& canBeCached, RuntimeMethodHandleInternal& ctor, Boolean& bNeedSecurityCheck)   at System.RuntimeType.CreateInstanceSlow(Boolean publicOnly, Boolean skipCheckThis, Boolean fillCache, StackCrawlMark& stackMark)   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean skipCheckThis, Boolean fillCache, StackCrawlMark& stackMark)   at System.Activator.CreateInstance(Type type, Boolean nonPublic)   at System.RuntimeType.CreateInstanceImpl(BindingFlags bindingAttr, Binder binder, Object[] args, CultureInfo culture, Object[] activationAttributes, StackCrawlMark& stackMark)   at System.Activator.CreateInstance(Type type, BindingFlags bindingAttr, Binder binder, Object[] args, CultureInfo culture, Object[] activationAttributes)   at System.Activator.CreateInstanceFromInternal(String assemblyFile, String typeName, Boolean ignoreCase, BindingFlags bindingAttr, Binder binder, Object[] args, CultureInfo culture, Object[] activationAttributes, Evidence securityInfo)   at System.AppDomain.CreateInstanceFrom(String assemblyFile, String typeName)System.IO.FileNotFoundException: Could not load file or assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies. The system cannot find the file specified. File name: 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'   at EFCorePowerTools.EFCorePowerToolsPackage.CreateServiceProvider()   at EFCorePowerTools.EFCorePowerToolsPackage..ctor() in C:\projects\efcorepowertools\src\GUI\EFCorePowerTools\EFCorePowerToolsPackage.cs:line 64 WRN: Assembly binding logging is turned OFF. To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1. Note: There is some performance penalty associated with assembly bind failure logging. To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].
--

### Steps to reproduce
Include a complete description that I can redo to reproduce the issue.

### Further technical details
EF Core Power Tools version: (found in About dialog - blue questionmark icon on context menu)
Version: 2.2.12

Database engine: (SQlite, SQL Compact, SQL Server, Postgres)

Visual Studio version:
Microsoft Visual Studio Community 2017
Version 15.9.7

**Comment by jespersh** [comment]
Could you check the version of netstandard.dll in these locations?
C:\Windows\Microsoft.NET\assembly\GAC_MSIL\netstandard\v4.0_2.0.0.0__cc7b13ffcd2ddd51\
C:\Windows\Microsoft.NET\Framework\v4.0.30319\
C:\Windows\Microsoft.NET\Framework64\v4.0.30319\

Should be version 4.7.xxxx. Also test with http://www.asoft.be/prod_netver.html - just to get that out of the way as a possible cause.
(danish locale)
![image](https://user-images.githubusercontent.com/1004198/53073688-05bff000-34e9-11e9-9efd-55504ce941d5.png)

**Comment by ErikEJ** [comment]
Do you have .NET 4.7.1 installed?

**Comment by vinayvhub** [comment]
> Could you check the version of netstandard.dll in these locations?
> C:\Windows\Microsoft.NET\assembly\GAC_MSIL\netstandard\v4.0_2.0.0.0__cc7b13ffcd2ddd51
> C:\Windows\Microsoft.NET\Framework\v4.0.30319
> C:\Windows\Microsoft.NET\Framework64\v4.0.30319\
> Should be version 4.7.xxxx. Also test with http://www.asoft.be/prod_netver.html - just to get that out of the way as a possible cause.
> (danish locale)


Thanks you very much for pointing me the asoft tool. That really helped to figure out the .NET version which I was using.

**Comment by vinayvhub** [comment]
> Do you have .NET 4.7.1 installed?

Thanks Erik, that was a problem, I had .NET 4.7 installed. I installed .NET 4.7.2 and all worked.

But I am facing new problem now. I used the option "Reverse engineer" to build dbcontext and models into my .NET CORE 2.2 Console application like you have showed in the demo. That part worked. After that I tried to add that colorful DbContext model diagram to my project then I go the below error in my output window. Need your input on how to solved this problem.

> ========== Build: 0 succeeded, 0 failed, 1 up-to-date, 0 skipped ==========

System.ArgumentException: Error:
System.ArgumentException: No EF Core DbContext types found in the project
   at ReverseEngineer20.EfCoreModelBuilder.GetDbContextTypes(DbContextOperations operations) in C:\Code\EFCorePowerTools\src\GUI\efpt\EFCoreModelBuilder.cs:line 76
   at ReverseEngineer20.EfCoreModelBuilder.BuildResult(String outputPath, Boolean generateDdl) in C:\Code\EFCorePowerTools\src\GUI\efpt\EFCoreModelBuilder.cs:line 34
   at ReverseEngineer20.Program.Main(String[] args) in C:\Code\EFCorePowerTools\src\GUI\efpt\Program.cs:line 44

Parameter name: processResult
   at EFCorePowerTools.Handlers.ModelAnalyzerHandler.Generate(String outputPath, Project project, GenerationType generationType) in C:\projects\efcorepowertools\src\GUI\EFCorePowerTools\Handlers\ModelAnalyzerHandler.cs:line 49
